src/viewport/index.tsx

```diff
@@ -245,6 +245,7 @@ class Viewport extends React.Component<ViewportProps, ViewportState> {
                         .getHexString();
                 colorIndex = colorIndex + 1;
             });
+	    this.visGeometry.finalizeIdColorMapping();
```

Comment:
this looks like it's got the wrong amount of indent

Reply:
It looks a bit funky but is in line w/ line 209 above (and passes the linter). The preceeding ~30 lines are a callback definition

Reply:
so is line 250 wrongly indented?